### PR TITLE
Add right-click menu items to copy Namespace URL and connection string

### LINKED ISSUE #320: Copy namespace URL/connection string action
In the navigation tree on the left, when you right-click a Queue or Topic, they have context menu items for copying their URLs, like "Copy Queue URL", "Copy Deadletter Queue URL", and so on. But when you click on the root namespace node, there's no options for copying URLs. I think it would be nice to have items for "Copy Namespace URL" and "Copy Connection String URI".

It would be convenient to have this, particularly for connections created using the multi-field dialogs in File > Connect, because you can't Save those, and as far as I can tell, they don't expose their connection string URIs anywhere else.

What do you think? If this is something you're okay with, I think I could code up a PR to do this.

### PR DESCRIPTION
**Pull request by apjanke**
Closes #320.

Adds right-click menu items at the root namespace node to copy its URL or connection string to the clipboard.

### CODE REVIEW

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.2.0-PullRequest0325.95.build.538 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/23077714) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/f3d5ff9e04 by @apjanke-ccs)

**Comment by SeanFeldman** [comment]
@apjanke we usually ask to provide a snapshot of the UI change or what a new thing would look like. Could you please add a comment with that? Thanks.

File: src/ServiceBusExplorer/Forms/MainForm.cs [hunk 1]
```diff
@@ -5326,6 +5326,36 @@ private async void ExportEntities(List<IExtensibleDataObject> list, string entit
...
+                    Clipboard.SetText(str);
+                }
+            }
+        }
+
+        private void copyNamespaceUrlMenuItem_Click(object sender, EventArgs e)
+        {
+            Uri uri = serviceBusHelper.NamespaceUri;
+            if (uri == null)
+                return;
```
Comment (SeanFeldman) [inline src/ServiceBusExplorer/Forms/MainForm.cs]: Surrounding curly braces for consistency.

File: src/ServiceBusExplorer/Forms/MainForm.cs [hunk 2]
```diff
@@ -5326,6 +5326,36 @@ private async void ExportEntities(List<IExtensibleDataObject> list, string entit
...
+                          ? uri.AbsoluteUri.Substring(0, uri.AbsoluteUri.Length - 1)
+                          : uri.AbsoluteUri;
+            copyStringToClipboard(prettyUri);
+        }
+
+        private void copyConnectionStringMenuItem_Click(object sender, EventArgs e)
+        {
+            var connectionString = serviceBusHelper.ConnectionString;
+            if (connectionString == null)
+                return;
```
Comment (SeanFeldman) [inline src/ServiceBusExplorer/Forms/MainForm.cs]: Surrounding curly braces for consistency.

**Comment by apjanke** [comment]
Here's a screenshot.

kehhedocbcdgkmfo

This PR adds two new “Copy Namespace URL” and “Copy Connection String” items to the namespace/root node context menu. For consistency with the other nodes’ context menus’ arrangement, they're positioned after the main Create/Delete section and before the Export/Import section.

File: src/ServiceBusExplorer/Forms/MainForm.cs (hunk 1 above)
Comment (apjanke) [reply src/ServiceBusExplorer/Forms/MainForm.cs]: Fixed.

File: src/ServiceBusExplorer/Forms/MainForm.cs (hunk 2 above)
Comment (apjanke) [reply src/ServiceBusExplorer/Forms/MainForm.cs]: Fixed.

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.2.0-PullRequest0325.99.build.542 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/23081514) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/7622bc8fa0 by @apjanke-ccs)

**Comment by apjanke** [comment]
I'm on a roll! We're starting a new Azure Queues project at work, so I'm using this as an opportunity to get familiar with SBE, which I expect we'll be using quite a bit.

**Comment by paolosalvatori** [comment]
Thanks @apjanke for yet another useful PR 🥇

**Pull request merged**